src/main/java/cam72cam/mod/item/Fuzzy.java

```diff
@@ -17,29 +17,35 @@
 
 /** OreDict / Tag abstraction.  Use for item equivalence */
 public class Fuzzy {
-    public static final Fuzzy WOOD_STICK = new Fuzzy(Tags.Items.RODS_WOODEN, "stickWood");
-    public static final Fuzzy WOOD_PLANK = new Fuzzy(ItemTags.PLANKS, "plankWood");
-    public static final Fuzzy REDSTONE_DUST = new Fuzzy(Tags.Items.DUSTS_REDSTONE, "dustRedstone");
+    public static final Fuzzy WOOD_STICK = new Fuzzy(Tags.Items.RODS_WOODEN, "stickWood").add(Items.STICK);
```

Comment:
Given that these use the existing minecraft tags, they should be imported automatically and not be required to be re-defined here.